scripts/push.sh

```diff
@@ -0,0 +1,4 @@
+set -ex
+
+pip install jina[hub]~=2.0
```

Comment:
I suggest using the latest master by default and make the Jina version an environment variable.

```suggestion
pip install jina[hub]~=2.0
```

Reply:
Ideally this should map to `jina` version of the executor itself

Reply:
This could be done with using dependabot, I think

Reply:
Diff PR though, once we decide on versioning